source/content/wordpress-cron.md

```diff
@@ -115,18 +167,27 @@ Replace `yourdomain.tld` with your domain.
 
 Depending on what service you use, you may have the ability to set up multiple jobs at different times. Creating a single job that calls your site's `wp-cron.php` script every 15 minutes is all you should need. WP-Cron will take care of the rest. If you create new schedules that need to be run more often than once every 15 minutes, you will need to adjust your job accordingly.
 
-It's important that you do not add a value to the `doing_wp_cron` query variable; it must be empty for the cron to work properly.
+It's important that you do not add a value to the `doing_wp_cron` query variable; it must be empty for the Cron to work correctly.
 
 ### Using Your Own Server
 
-If you administer your own server, you can use the cron service to make a call to the `wp-cron.php` script. You will have to learn how to properly set up a Cron job and use something like `wget` or `curl` to fetch a web page. Unless you take special precautions, it is not any safer to use your own server vs. a web-based cron service; however, it does give you more control.
+If you administer your own server, you can use the Cron service to make a call to the `wp-cron.php` script. You will have to learn how to properly set up a Cron job and use something like `wget` or `curl` to fetch a web page. Unless you take special precautions, it is not any safer to use your own server vs. a web-based Cron service; however, it does give you more control.
 
-### Security
+### WordPress Cron Plugins
+
+WordPress has many plugins that control both internal WP-Cron tasks and external Cron jobs. Since Pantheon does not provide Cron services, we do not recommend or discourage the use of any given plugin. We encourage you to check out the list of [WordPress Cron Plugins](https://wordpress.org/plugins/search.php?q=cron "List of WordPress plugins that help manage cron jobs") and experiment in your Dev environment to find the one that best suits your needs.
 
-For the most part, `wp-cron.php` is secure. There is only one parameter you can pass in that will affect the script (`doing_wp_cron`). This `$_GET` value is not filtered; it is only used as a flag and not as input for a process or variable. Beyond that, all input is ignored.
+## Troubleshooting WP-Cron
```

Comment:
Now that Pantheon Cron is calling WordPress cron once an hour, the troubleshooting section should be adjusted.

For example, low-traffic sites might still have problems if they receive absolutely no traffic for twelve hours, at which point they might be put to sleep, and cron jobs will stop. Until that happens, though, cron will continue to run every hour, regardless of whether there was any traffic in any given hour.

Reply:
High traffic sites should experience fewer problems if they just disable WP Cron and allow Pantheon cron to handle periodic tasks. In the future, we will do this by default for all WordPress sites, and users will need to re-enable WP-Cron if they want it to keep running.